src/server/campaignd/server.cpp

```diff
@@ -769,10 +765,8 @@ void server::handle_request_campaign(const server::request& req)
 	auto version_map = get_version_map(campaign);
 
 	if(version_map.empty()) {
-		// Old format ?
-		ERR_CS << "The (" + to + ") version of the addon '" << req.cfg["name"].str()
-			   << "' is stored in the old format (bug?)! Trying to send a legacy full-pack.";
-		to = campaign["version"].str();
+		send_error("No versions of the addon '" + req.cfg["name"].str() + "' have been found by the server!", req.sock);
```

Comment:
Quoting myself from Discord:

> "No versions of the addon 'foo' have been found by the server!" really should be "No versions of the add-on 'foo' are available on the server." in my opinion

(CC @Vultraz)

Reply:
Changed the line wording, Vultraz approved it in a Discord message.